browser/ui/views/split_view/split_view_separator.cc

```diff
@@ -25,13 +109,52 @@ bool SplitViewSeparator::OnMousePressed(const ui::MouseEvent& event) {
   return ResizeArea::OnMousePressed(event);
 }
 
+void SplitViewSeparator::Layout(PassKey) {
+  LayoutMenuButton();
+}
+
 void SplitViewSeparator::OnResize(int resize_amount, bool done_resizing) {
   // When mouse goes toward web contents area, the cursor could have been
   // changed to the normal cursor. Reset it resize cursor.
   GetWidget()->SetCursor(ui::Cursor(ui::mojom::CursorType::kEastWestResize));
   if (resize_area_delegate_) {
     resize_area_delegate_->OnResize(resize_amount, done_resizing);
   }
+  LayoutMenuButton();
```

Comment:
While resizing, I could see resize bar and menu buttton(bubble) are not synchronously moved.
How about hiding menu button while resizing? or could move both synchronously?

Reply:
> How about hiding menu button while resizing?

That's a good idea!